v3/how-to-guides/04-ocw/b-http-requests/index.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Making HTTP Requests
+slug: /how-to-guides/v3/ocw/http-requests
+keywords: ['off-chain worker', 'ocw', 'http', 'https', 'requests']
+version: '3.0'
+section: how to guides
+category: ocw
+---
+
+Making HTTP Requests
+
+<Objectives data={[
+  {
+    title: 'Goal',
+    description: `Send http requests offchain`
+  },
+  {
+    title: 'Use Cases',
+    description: `GET data from offchain, or POST data offchain.`,
+  },
+]}/>
+
+## Overview
+
+The offchain-worker (**ocw** for short)  was introduced to simplify the building of oracles. Because a blockchain does not have access to data outside its own network, an oracle is required to provide context on decision Making
+
```

Comment:
```suggestion
Off-chain workers (**OCW** for short) were introduced to extend oracle-like capabilities for Substrate blockchains. 
Because a blockchain does not have access to data outside its own network, oracles are a useful tool to enable interactions between on and off-chain worlds.

```